adapters/Facebook/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Facebook iOS Mediation Adapter Changelog
 
+#### Version 5.8.0.0
+- Verified compatibility with FAN SDK 5.8.0.
```

Comment:
Please list this under the `Next Version` below, and update it to `Version 5.8.0.0`.

Reply:
You refer to write it like this:

#### Next Version
- Verified compatibility with FAN SDK 5.8.0.

 where should i add the Version 5.8.0.0?

Reply:
Hi @yroxe 

Apologies for the confusion. I was referring to write it like this, since the previous release note tagged as `Next Version` is for the next adapter release:

```
#### Version 5.8.0.0
- Verified compatibility with FAN SDK 5.8.0.
- Adapter now returns a non-zero `mediaContent` aspect ratio regardless if the media view is rendered or not.
```

Hope this clears up the confusion. Thank you!

Reply:
Hi Joshua,

No worries, thank you, I changed the text. If anything else is needed, please let me know.